cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go

```diff
@@ -147,6 +147,15 @@ func (t Template) controllersPolicy() *iamv1.PolicyDocument {
 					"secretsmanager:TagResource",
 				},
 			},
+			{
```

Comment:
After you've rebased, this should go behind the EKS flags.

Reply:
Any precedence for feature flags in clusterawsadm yet?

Reply:
Sorry, misspoke. I meant the flag in terms of `t.Spec.ClusterAPIControllers.EKS.Enable` at https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go#L149 than a `-- features` thing

Reply:
done